rpc_jobs/periodic_cleanup.yml

```diff
@@ -82,10 +82,28 @@
                   sh "sudo restart vpnc-kronos"
                 }
                 common.docker_cache_workaround()
-                String sha = sh(script: "git rev-parse --verify HEAD", returnStdout: true).trim()
                 // Docker version on master doesn't like '/' in the name, using '-' as the separator
-                name = docker_utils.toInternalRegistryName("periodic_cleanup", sha, "re", env.RE_JOB_REPO_NAME, "-")
-                container = docker_utils.pullOrBuild(name, null)
+                name = docker_utils.toInternalRegistryName("periodic-cleanup", "latest", "re", env.RE_JOB_REPO_NAME, "-")
+                // jenkins docker plugin stuff does not work
```

Comment:
`docker.image` and `container.inside` are used below, presumably they are included in "docker plugin stuff"?

Reply:
It turns out this job has never been using a container, it always fails with only a log entry to indicate there was an issue and then runs the code on the enclosing host. I copied that section from the build_summary job and didn't spot the inconsistency with you comment referencing stuff. I'm going to push a patch to update this to remove the use of the Docker plugin in the job.

Reply:
https://rpc.jenkins.cit.rackspace.net/job/Periodic-Cleanup/1638 shows a run using the code from this pull request.